cmd/storagenode-updater/testdata/servicebuilder/main.go

```diff
@@ -0,0 +1,38 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strconv"
+)
+
+var (
+	exitCode string
```

Comment:
Can you add a comment to these 2 vars that their values must be set by `build ldflags`?
It wasn't clear to me until I saw where this is built.